Mathlib/AlgebraicGeometry/Modules/Tilde.lean

```diff
@@ -143,7 +143,7 @@ See also `Tilde.isLocallyFraction`.
 def tildeInType : Sheaf (Type u) (PrimeSpectrum.Top R) :=
   subsheafToTypes (Tilde.isLocallyFraction M)
 
-instance (U : (Opens (PrimeSpectrum.Top R))ᵒᵖ) :
+noncomputable instance (U : (Opens (PrimeSpectrum.Top R))ᵒᵖ) :
```

Comment:
Why was this not needed before?

Reply:
I think it got smarter because the instance was better structured. The next one is already `noncomputable` for the same one (probably) this is now.